src/components/organelle-wrapper.tsx

```diff
@@ -0,0 +1,101 @@
+import * as React from 'react';
+declare var Organelle: any;
+
+interface OrganelleWrapper {
+  name: string,
+  modelProperties: any,
+  showBinding: boolean,
+  doAddHormone: boolean,
+  addEnzyme: boolean,
+  setActiveAssay: Function,
+  currentView: any,
+  setGraphState: Function
+}
+
+class OrganelleWrapper extends React.Component<any, any> {
+    model: any;
+  constructor(props: OrganelleWrapper) {
+    super(props);
+    this.model = null;
+    this.addHormone = this.addHormone.bind(this);
+  }
+
+  componentDidMount() {
+    const {modelProperties} = this.props;
+
+    Organelle.createModel({
+      element: this.props.name,
+      background: {
+        file: "assets/melanocyte.svg",
+        selector: "#cell"
+      },
+      properties: modelProperties,
+      calculatedProperties: {},
+      clickHandlers: [
+        {
+          selector: "#melanocyte_x5F_cell",
+          action: this.organelleClick.bind(this, "cytoplasm")
+        },
+        {
+          selector: "#nucleus_x5F_A",
+          action: this.organelleClick.bind(this, "nucleus")
+        },
+      ],
+      species: [
```

Comment:
FWIW, I figured out why melamosomes didn't always appear in `mlm-demos` - model species are only set in `componentDidMount`, so melamosomes only appear if `golgi` was the first selected view. Is there an easy way to update the species in a model mid-run?